### PR TITLE
Make more the default stack size to DEFAULT_TASK_STACKSIZE

### PR DESCRIPTION
**Pull request by xiaoxiang781216**

### CODE REVIEW

File: examples/mount/Kconfig [hunk 1]
```diff
@@ -35,7 +35,7 @@ if !EXAMPLES_MOUNT_BLOCKDEVICE
 
 config EXAMPLES_MOUNT_NSECTORS
 	int "RAM disk number of sectors"
-	default DEFAULT_TASK_STACKSIZE
+	default 2048
```
Comment (Ouss4) [inline examples/mount/Kconfig:38]: Isn't that supposed to be the opposite way?

Comment (Ouss4) [reply examples/mount/Kconfig:38]: Never mind, I see it has nothing to do with a task.

Comment (xiaoxiang781216) [reply examples/mount/Kconfig:38]: This is a wrong change by:
commit e7156be066c7427e2afb56521c3eaf56d7623e25
Author: YAMAMOTO Takashi <yamamoto@midokura.com>
Date:   Fri Mar 27 14:10:33 2020 +0900

    Change the defaults of stack size configs to DEFAULT_TASK_STACKSIZE
    
    This commit changes only ones with the default 2048 and
    leaves the others.
    E.g. this leaves SYSTEM_RAMTEST_STACKSIZE, whose default is 1024.
    I guess those need to be inspected one-by-one.

**Comment by xiaoxiang781216** [comment]
Need merge https://github.com/apache/incubator-nuttx/pull/872 fist to pass the check.

**Comment by Ouss4** [comment]
There seems to be an issue with the checks (or Github), I restarted both this PR and nuttx#872 a couple of times, they always fail somewhere.
Maybe we need @liuguo09 or @btashton inputs' for that.

**Comment by btashton** [comment]
The failures look like this and I think it's because the two patches are dependent on each other. Now that we have the branch logic in for the releases, it will be fairly easy to add the PR linking logic, but I was waiting to get the release out first.

I would be fine merging the app side in now and then making sure the OS pr passes. 

```
Building NuttX... 

Normalize lc823450-xgevk/audio 
file1: CONFIG_SYSTEM_USBMSC_CMD_STACKSIZE=2048 

file2: 

Saving the new configuration file 

On branch master 

Your branch is up to date with 'origin/master'. 

Changes not staged for commit: 
(use "git add <file>..." to update what will be committed) 
(use "git checkout -- <file>..." to discard changes in working directory) 

modified: boards/arm/lc823450/lc823450-xgevk/configs/audio/defconfig 

no changes added to commit (use "git add" and/or "git commit -a")

```

File: graphics/nxwm/Kconfig [hunk 2]
```diff
@@ -504,7 +504,7 @@ config NXWM_TOUCHSCREEN_LISTENERPRIO
 
 config NXWM_TOUCHSCREEN_LISTENERSTACK
 	int "Touchscreen Listener Task Stack Size"
-	default 1024
```
Comment (patacongo) [inline graphics/nxwm/Kconfig:507]: This changes the default stack size

File: netutils/discover/Kconfig [hunk 3]
```diff
@@ -15,7 +15,7 @@ if NETUTILS_DISCOVER
 
 config DISCOVER_STACK_SIZE
 	int "Discover Daemon Stack Size"
-	default 1024
```
Comment (patacongo) [inline netutils/discover/Kconfig:18]: This changes the default stack size

File: system/hex2bin/Kconfig [hunk 4]
```diff
@@ -24,7 +24,7 @@ if SYSTEM_HEX2BIN_BUILTIN
 
 config SYSTEM_HEX2BIN_STACKSIZE
 	int "hex2bin stack size"
-	default 1536
```
Comment (patacongo) [inline system/hex2bin/Kconfig:27]: This changes the default stack size

File: system/hex2bin/Kconfig [hunk 5]
```diff
@@ -75,7 +75,7 @@ if SYSTEM_HEX2MEM_BUILTIN
 
 config SYSTEM_HEX2MEM_STACKSIZE
 	int "hex2mem stack size"
-	default 1536
```
Comment (patacongo) [inline system/hex2bin/Kconfig:78]: This changes the default stack size

File: system/lm75/Kconfig [hunk 6]
```diff
@@ -32,7 +32,7 @@ endchoice
 
 config SYSTEM_LM75_STACKSIZE
 	int "Stack size (bytes)"
-	default 1024
```
Comment (patacongo) [inline system/lm75/Kconfig:35]: This changes the default stack size

File: system/nxplayer/Kconfig [hunk 7]
```diff
@@ -21,7 +21,7 @@ config NXPLAYER_MAINTHREAD_STACKSIZE
 
 config NXPLAYER_PLAYTHREAD_STACKSIZE
 	int "NxPlayer thread stack size"
-	default 1500
```
Comment (patacongo) [inline system/nxplayer/Kconfig:24]: This changes the default stack size

File: system/nxrecorder/Kconfig [hunk 8]
```diff
@@ -15,7 +15,7 @@ if SYSTEM_NXRECORDER
 
 config NXRECORDER_RECORDTHREAD_STACKSIZE
 	int "NxRecorder thread stack size"
-	default 1500
```
Comment (patacongo) [inline system/nxrecorder/Kconfig:18]: This changes the default stack size

File: system/tee/Kconfig [hunk 9]
```diff
@@ -13,7 +13,7 @@ if SYSTEM_TEE
 
 config SYSTEM_TEE_STACKSIZE
 	int "tee stack size"
-	default 1536
```
Comment (patacongo) [inline system/tee/Kconfig:16]: This changes the default stack size

File: system/usbmsc/Kconfig [hunk 10]
```diff
@@ -156,7 +156,7 @@ endif # SYSTEM_USBMSC_TRACE
 
 config SYSTEM_USBMSC_CMD_STACKSIZE
 	int "Stacksize of msconn and msdis commands"
-	default 768
```
Comment (patacongo) [inline system/usbmsc/Kconfig:159]: This is a horrendous increate in the default stack size

**Comment by patacongo** [comment]
After reviewing an trying to comment on all of the problems and risks.  I come to the conclusion that this change is wrong and must not be merged.

**Comment by patacongo** [comment]
Although this change is improper and must not be merged, there are also two important bugfixes in the change.  I pulled those out and submitted them separately as PR204.

**Pull request closed without merging**